packages/parser/test/parse.test.ts

```diff
@@ -283,15 +381,22 @@ describe('Tokens', () => {
       [
         'invalid: Gradient (bad syntax)',
         {
-          given: {
-            gradient: {
-              $type: 'gradient',
-              $value: [{ color: '{color.blue.500', position: '{perc.0}' }],
+          given: [
+            {
+              filename: DEFAULT_FILENAME,
+              src: {
+                gradient: {
+                  $type: 'gradient',
+                  $value: [{ color: '{color.blue.500', position: '{perc.0}' }],
+                },
+              },
             },
-          },
+          ],
           want: {
             error: `Invalid alias: "{color.blue.500"
 
+/tokens.json:6:18
```

Comment:
This is the real magic: in error outputs, it preserves what file the error originated from, and also specifies line/column.